test/supplier.test.js

```diff
@@ -67,9 +110,27 @@ describe('Suppliers', () => {
                 .set('authorization', credentials.token)
                 .send({PostBody: {filter: {}, structure: '*', limit: 20, page: 1}});
             for (const element of res.body.datas) {
-                const deleteOne = await chai.request(app).delete(`/api/v2/supplier/${element._id}`).set('authorization', credentials.token);
+                const deleteOne = await chai.request(app)
+                    .delete(`/api/v2/supplier/${element._id}`)
+                    .set('authorization', credentials.token);
                 expect(deleteOne).to.have.status(200);
             }
         });
+        it('Try delete a supplier - w/o authentication', async () => {
+            const supplier = await createSuppliers();
+            const res      = await chai.request(app)
+                .delete(`/api/v2/supplier/${supplier._id}`);
+            expect(res).to.have.status(401);
+            expect(res.body).have.property('code');
+            expect(res.body.code).to.be.equal('Unauthorized');
+        });
+        it('Try delete a supplier - w/o the good ID', async () => {
+            await createSuppliers();
+            const res = await chai.request(app)
+                .delete('/api/v2/supplier/111111111111111111111111')
+                .set('authorization', credentials.token);
+            expect(res).to.have.status(200);
+            // TODO fix services suplliers to not send back 200 but use 'findOneAndDelete'
```

Comment:
Is this a normal comportement ?
@devaquila 

(check the return of `deleteSupplier` in `services/suppliers.js:33`)